### ISSUE TITLE
Support node labels

### ISSUE THREAD

**Issue by sananguliyev** [vitobotta/hetzner-k3s#98]
Firstly, thanks a lot for the nice project! Is there any plan to support custom label or annotations for servers, since it's very useful for separating the deployments between nodes. There are a bunch of default labels like instance type, location, hostname but sometime devs need to have some custom labels, too.

**Comment by Privatecoder** [comment]
Does this need to happen during deployment? I add additional labels and taints aftrewards.

**Comment by vitobotta** [comment]
Hi @sananguliyev, the tool doesn't add any labels to the nodes. What kind of labels where you thinking?

**Comment by sananguliyev** [comment]
Thanks for your answer. Let's assume that I am using the same type of servers for 2 different purposes. 
1.  API gateway
2. SQL Engine

It's very useful to add labels to nodes so when I deploy my gateway I can specify in my k8s deployment that run this app only on the nodes which has labelled `api-gateway`.

**Comment by vitobotta** [comment]
I understand. However I think this goes beyond the scope of the tool which is creating clusters with a common configuration. What you are asking is for specific use cases. Would you want to add the same labels on all the nodes in a pool or how?

**Comment by sananguliyev** [comment]
Yes, exactly. I would like to have those labels on all nodes in the pool, otherwise it would be too specific. 

Desired configuration:

```
worker_node_pools:
- name: sql-engine-small
  instance_type: cpx21
  instance_count: 3
  labels: 
    purpose: sql-engine
    size: small
```

**Comment by vitobotta** [comment]
Ok, I'll see what I can do.

**Comment by sananguliyev** [comment]
Perfect. Thanks in advance.

**Comment by Privatecoder** [comment]
@vitobotta if you add this you can also go ahead and add the same for taints, i.e.
```
worker_node_pools:
- name: sql-engine-small
  instance_type: cpx21
  instance_count: 3
  labels: 
    purpose: sql-engine
    size: small
  taints: 
    key: value
```

**Comment by vitobotta** [comment]
Ok

**Comment by vitobotta** [comment]
Implemented in v0.6.2. Please give it a try and let me know how it goes.
